### PR TITLE
add url changes detector and launch accordingly

### PR DESCRIPTION
**Pull request by antoinebollinger**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -10,17 +10,23 @@
...
     "activeTab",
-    "storage"
+    "storage",
+    "tabs"
   ],
   "content_scripts": [
     {
       "matches": [
-        "https://openclassrooms.com/fr/dashboard/*"
+        "https://openclassrooms.com/*"
```
Comment (fabrahaingo) [inline manifest.json]: I don't think running the extension on all pages of `openclassrooms.com` is necessary here.
What do you think of
```javascript
[
"https://openclassrooms.com/fr/dashboard/*", // french version
"https://openclassrooms.com/en/dashboard/*" // english version
]
```

File: src/main.js [hunk 2]
```diff
@@ -47,4 +47,15 @@ async function main() {
   createSessionsTable(sessionsList, anchor)
 }
 
-main()
+// Indicator if table has already bean build
```
Comment (fabrahaingo) [inline src/main.js:50]: Typo here: `hear` instead of `here` ;)

File: src/main.js [hunk 3]
```diff
@@ -47,4 +47,15 @@ async function main() {
...
+let done = false;
+
+// Check URL changing and launch main() if we're on Sessions & done == false, i.e. not build yet
```
Comment (fabrahaingo) [inline src/main.js:53]: Typo here: `build` instead of `built`

**Review by fabrahaingo — CHANGES_REQUESTED** [review]
2 typos + 1 optimization and it's good to go 👍

File: manifest.json (hunk 1 above)
Comment (antoinebollinger) [reply manifest.json]: I agree. Solution tested and working on local.

File: src/main.js (hunk 2 above)
Comment (antoinebollinger) [reply src/main.js:50]: I cannot see "hear" in this comment. Are you talking about "bean" that should be "been" ?

File: src/main.js (hunk 3 above)
Comment (antoinebollinger) [reply src/main.js:53]: Yep!

**Pull request merged**